src/components/SettingsDialog/ProfileSettings/ProfileSettings.scss

```diff
@@ -1,35 +1,71 @@
+@import "src/constants/style";
 @import "../SettingsDialog";
 
-.profile-settings__user-name-button {
-  justify-content: space-between;
-  cursor: text;
-}
+.profile-settings {
+  &__container {
+    display: flex;
+    flex-direction: column;
+    gap: $margin--large;
+    height: 100%;
+    width: 100%;
+    max-width: calc(100vw - (2 * #{$settings-dialog-container--sides-small}));
 
-.profile-settings__user-name-button span {
-  cursor: inherit;
-}
+    @media screen and (min-width: 450px) {
+      max-width: calc(380px - (2 * #{$settings-dialog-container--sides-small}));
+    }
 
-.profile-settings__user-name-button_input {
-  background-color: transparent;
-  border: none;
-  padding: $padding--default $padding--medium $padding--default $padding--default;
-  font-weight: bold;
-  letter-spacing: $letter-spacing--medium;
-  font-size: $text-size--medium;
-  text-align: right;
-  width: 60%;
-  cursor: inherit;
-}
+    @media screen and (min-width: 920px) {
+      max-width: calc(544px - (2 * #{$settings-dialog-container--sides-large}));
```

Comment:
Where do the `544px` come from?

Reply:
Tbh idk, but you should 🤓 . I reused it from the `SettingsDialog` and `BoardSettings` which implement the scrollbar.

![image](https://user-images.githubusercontent.com/44020029/174991233-8abee172-88ac-4209-a03f-4fbe454991e3.png)

![image](https://user-images.githubusercontent.com/44020029/174991283-21b46e0d-f020-492a-a80d-76ee55673cbf.png)